pastastore/store.py

```diff
@@ -982,9 +982,9 @@ def get_model_timeseries_names(
                     enam = sm["evap"]["name"]
                     structure.loc[mlnam, pnam] = 1
                     structure.loc[mlnam, enam] = 1
-                else:
-                    for s in sm["stress"]:
-                        structure.loc[mlnam, s["name"]] = 1
+                elif "stress" in sm:
```

Comment:
@martinvonk I don't think this will work when stress is a list, e.g. for WellModel. I'm assuming you needed this change for something recent, can you add something that will work in both cases?

Reply:
I just had an issue where "stress" was not in the sm dict..

Reply:
see latest commit.

Reply:
:+1: